player/loadfile.c

```diff
@@ -61,6 +61,13 @@
 #include "core.h"
 #include "command.h"
 
+// This is the same list as in etc/mpv.desktop, keep them in sync. (mpv:// is not included.)
+static const char *const safe_protocols[] = {
+    "appending", "bd", "cdda", "dvb", "file", "ftp", "hls", "http",
+    "https", "mms", "rist", "rtmp", "rtmps", "rtmpt", "rtmpts", "rtp",
+    "rtsp", "rtsps", "sftp", "smb", "srt", "srtp", "webdav", "webdavs",
+};
```

Comment:
Can we merge this list with what's in `stream_lavf` somehow? Feels like duplication.

Reply:
I changed some things.